infrastructure/lib/stacks/monitoringDashboard.ts

```diff
@@ -80,7 +80,7 @@ export class OpenSearchMetricsMonitoringStack extends Stack {
     private snsMonitorCanaryFailed(canaryName: string, canaryUrl: string, vpcStack: VpcStack): void {
         const canary = new Canary(this, 'CanaryHeartbeatMonitor', {
             canaryName: canaryName,
-            schedule: Schedule.rate(Duration.minutes(1)),
+            schedule: Schedule.rate(Duration.minutes(15)),
```

Comment:
Maybe we should do once every 5 minutes, since the test is rarely but occasionally flaky(running once a minute it fails due to the UI taking too long to load about once every few days), I would set the threshold to be lower than 100, and have the evaluation period to be a few times more than how often the canary runs. I say we know the canary is actually failing if it fails 3 times in a row, which if it runs every 5 minutes, we will know in 15 minutes.

Reply:
I see let me update the `Schedule.rate` to 5 mins.